tests/test_client_side_cache.py

```diff
@@ -177,6 +177,7 @@ async def test_unsafe_redis_down(client_side_listen_timeout: Optional[float]):
 
     cache = BcastClientSide(
         client_side_listen_timeout=client_side_listen_timeout,
+        suppress=False,
```

Comment:
This is the breaking change introduced by the PR -- Redis client side would previously raise on initialization failures/timeouts.